saw-core/src/Verifier/SAW/Term/Pretty.hs

```diff
@@ -20,6 +20,7 @@ module Verifier.SAW.Term.Pretty
   , depthPPOpts
   , ppNat
   , ppTerm
+  , ppTermInCtx
```

Comment:
Is `ppTermInCtx` something that would make sense in the master branch? Maybe this should get its own PR.

Reply:
Sure, that makes sense. It just exposes a little more power in the pretty-printing API, nothing that I imagine would break any existing code.

Reply:
See #167.